Formula/mono.rb

```diff
@@ -52,22 +58,20 @@ class Mono < Formula
       url "https://raw.githubusercontent.com/mono/mono/a22ed3f094e18f1f82e1c6cead28d872d3c57e40/packaging/MacSDK/patches/fsharp-netfx-multitarget.patch"
       sha256 "112f885d4833effb442cf586492cdbd7401d6c2ba9d8078fe55e896cc82624d7"
     end
-    patch do
-      url "https://github.com/dotnet/fsharp/commit/be6b22d11ae996b2d9b8e0724d9cf05ad65a0485.patch?full_index=1"
-      sha256 "793a39da798673b99289f3ac344ff8bd23d7eea2d3366c28e7e42229d8b130ca"
-    end
   end
 
-  resource "fsharp-layout-patch" do
-    url "https://raw.githubusercontent.com/mono/mono/3070886a1c5e3e3026d1077e36e67bd5310e0faa/packaging/MacSDK/fsharp-layout.sh"
-    sha256 "f2cc63bf77e50663d91c6d102ba1d9217d1b9100c57071f79f0ae5a45e80ef42"
-  end
-
-  # When upgrading Mono, make sure to use the revision from
-  # https://github.com/mono/mono/blob/mono-#{version}/packaging/MacSDK/msbuild.py
   resource "msbuild" do
-    url "https://github.com/mono/msbuild.git",
-        revision: "70bf6710473a2b6ffe363ea588f7b3ab87682a8d"
+    # When upgrading Mono, make sure to use the revision from
+    # https://github.com/mono/mono/blob/mono-#{version}/packaging/MacSDK/msbuild.py
+    on_macos do
+      url "https://github.com/mono/msbuild/archive/63458bd6cb3a98b5a062bb18bd51ffdea4aa3001.tar.gz"
+      sha256 "f0c5f376a82ce9c4468f53374833ac0d8f4d90749aaaed5747e46760ffc679f3"
+    end
+    # Currently selecting latest release from repo.
+    on_linux do
+      url "https://github.com/mono/linux-packaging-msbuild/archive/refs/tags/upstream/16.10.1+xamarinxplat.2021.05.26.14.00.tar.gz"
+      sha256 "73c948c086d8cea2914917f3b34c084948e734bdb2286722a7c07eeeb59cffed"
+    end
```

Comment:
Maybe will switch to official tarball in next run:
```suggestion
    on_linux do
      url "https://download.mono-project.com/sources/msbuild/msbuild-16.10.1+xamarinxplat.2021.05.26.14.00.tar.xz"
      sha256 "2921f3f66ad96c2e1e90e174a9cc1cf29717e56fb39e3b3b684313d926c6f015"
    end
```